src/main/java/org/example/zad3.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class zad3 {
+
+    static Scanner in = new Scanner(System.in);
```

Comment:
Add new line after the variable. Add new line after "static Scanner in = new Scanner(System.in);".